.travis-scripts/docs.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash -e
+
+run() {
+	printf "$ $ANSI_YELLOW%s$ANSI_CLEAR\n" "$(echo $@)" >&2
+	"$@"
+}
+
+remote_url="$(git remote get-url origin)"
+pages_dir="$TRAVIS_BUILD_DIR-gh-pages"
+
+echo 'Write GITHUB_ACCESS_TOKEN to ~/.netrc'
+cat > ~/.netrc << EOF
+machine github.com
+  login $GITHUB_ACCESS_TOKEN
+EOF
+
+run git clone --depth=1 --no-single-branch --branch=gh-pages "$remote_url" "$pages_dir"
+run cd "$pages_dir"
+
+run find . -name .is_ref | while read line; do
+	ref="$(dirname "$line")"
+	ref="${ref:2}"
+	if ! git rev-parse --verify -q "refs/remotes/origin/$ref" > /dev/null; then
+		run rm -rf "$ref"
+	fi
+done
+
+run git status
+
+run cd "$TRAVIS_BUILD_DIR"
+run docker run --rm -i -v "$TRAVIS_BUILD_DIR:/app" -w /app mgjm/sn3t:base << EOF
```

Comment:
Here as well.

Reply:
The docker images are not available on the OSM docker-hub page right now because of the build pipeline being setup at the moment.